src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
@xaiki That is fully a mistake on my end, I forgot to switch branches when making these changes (I was still pushing to patch-2) when I was meaning to submit to different PR with these changes :smile: 

Sorry for any inconveniences that have been caused